README.md

```diff
@@ -1 +1,287 @@
+AAPL Quote: {AAPL, AskExchangeCode: 'K', Ask: 4@144.96, AskTime: 2021-07-29T12:47:28.0000000Z, BidExchangeCode: 'K', Bid: 2@144.93, BidTime: 2021-07-29T12:47:29.0000000Z, Scope: Composite}
+```
+
+---
+
+## Samples
+
+[https://github.com/dxFeed/dxfeed-net-api/tree/master/samples](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples)
+
+  * [dxf_candle_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_candle_sample) - demonstrates how to subscribe to **`Candle`** event.
+  * [dxf_events_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_events_sample) - demonstrates how to subscribe to **`Quote`**, **`Trade`**, **`TradeETH`**, **`Order`**, **`SpreadOrder`**, **`Profile`**, **`Summary`**, **`TimeAndSale`**, **`Underlying`**, **`TheoPrice`**, **`Series`**, **`Greeks`**, **`Configuration`** events.
+  * [dxf_inc_order_snapshot_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_inc_order_snapshot_sample) - demonstrates how to subscribe to order snapshot with incremental updates.
+  * [dxf_instrument_profile_live_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_instrument_profile_live_sample) - demonstrates how to 
+  * [dxf_instrument_profile_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_instrument_profile_sample) - demonstrates how to 
+  * [dxf_ipf_connect_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_ipf_connect_sample) - demonstrates how to 
+  * [dxf_option_chain_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_option_chain_sample) - demonstrates how to 
+  * [dxf_order_view_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_order_view_sample) - demonstrates how to 
+  * [dxf_order_view_xcheck](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_order_view_xcheck) - demonstrates how to 
+  * [dxf_price_level_book_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_price_level_book_sample) - demonstrates how to 
+  * [dxf_promises_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_promises_sample) - demonstrates how to 
+  * [dxf_read_write_raw_data_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_read_write_raw_data_sample) - demonstrates how to save/read incoming binary traffic to/from file.
+  * [dxf_regional_book_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_regional_book_sample) - demonstrates how to 
+  * [dxf_simple_order_book_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_simple_order_book_sample) - demonstrates how to 
+  * [dxf_snapshot_sample](https://github.com/dxFeed/dxfeed-net-api/tree/master/samples/dxf_snapshot_sample) - demonstrates how to subscribe to **`Order`**, **`SpreadOrder`**, **`Candle`**, **`TimeAndSale`**, **`Greeks`**, **`Series`** snapshots.
```

Comment:
`dxf_simple_data_retrieving_sample` has also been added. It is worth mentioning it too.